client/me/security-u2f-key/index.jsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
+ */
+import Gridicon from 'gridicons';
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { isSupported } from 'u2f-api';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import SecurityU2fKeyAdd from './add';
+import SecurityU2fKeyList from './list';
+import { recordGoogleEvent } from 'state/analytics/actions';
+import wpcom from 'lib/wp';
+
+class SecurityU2fKey extends React.Component {
+	static initialState = Object.freeze( {
```

Comment:
If you're worried about mutation, it may be better to return a new object from a function when you need to get a clean initial state.

As is, this is an Object containing other Objects, and the inner objects can be mutated.

Reply:
I have no idea why this was done.